plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVmMemoryDeviceDef.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 The Apache Software Foundation.
```

Comment:
@GutoVeronezi I think that these source code headers need to be updated removing  **Copyright** as we discussed in another PR (sorry but I forgot which one xD)